### DISCUSSION TITLE
posts/cilium-amazon-eks/

### DISCUSSION THREAD

**Discussion by giscus** [ruzickap/ruzickap.github.io#260] [General]
# posts/cilium-amazon-eks/

Build “cheap and secure” Amazon EKS with Karpenter and Cilium

https://ruzickap.github.io/posts/cilium-amazon-eks/

**Comment by kkumtree** [comment]
Thanks for posting about EKS and cilium intergration. I really want to know the guide like this. I hope this post will be my guidance to use cilium. :)

**Reply by ruzickap** [reply]
I'm glad it is useful...
Thank you.
